### ISSUE TITLE
Multilevel conditions "Add condition" button not working and throwing this.suggestableFields console error.

### ISSUE THREAD

**Issue by minemindmedia** [statamic/cms#9578]
### Bug description

I was attempting to add conditions to some fields within a grid field and the "add condition" button does not add a condition and throws a console error.

### How to reproduce

Install new Statamic project.
Open page blueprint.
Add grid field.
Add field within grid field.
Attempt to apply condition to field.

Add condition button won't open new condition options and throws console error `this.sugestableFields`.

### Environment

```yaml
Environment
Application Name: Statamic
Laravel Version: 10.45.1
PHP Version: 8.2.12
Composer Version: 2.5.5
Environment: local
Debug Mode: ENABLED
URL: statamic.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 0
Antlers: runtime
Sites: 1
Stache Watcher: Enabled
Static Caching: Disabled
Version: 4.49.0 Solo
```

### Installation

Fresh statamic/statamic site via CLI

### Antlers Parser

None

### Additional details

Work properly with v4.32
Have not tested between 4.32 and 4.49

**Comment by jasonvarga** [comment]
I haven't confirmed, but if I was to guess, it's probably introduced by #9379.

**Comment by duncanmcclean** [comment]
Yep, that was it! Fixed in #9586.
